geometry/list-transformations-by-suitability/README.md

```diff
@@ -27,6 +27,22 @@ If the selected transformation is not usable (has missing grid files) then an er
 * GeometryEngine
 * TransformationCatalog
 
+## Additional information
+
+Some transformations aren't available until transformation data is provided.
+
+This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTrasnformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
+
+This sample can be used with or without provisioning projection engine data to your device. If you do not provision data, a limited number of transformations will be available.
+
+To download projection engine data to your device:
+
+1. Login to the [ArcGIS for Developers site](https://developers.arcgis.com/sign-in/) using your Developer account.
+2. In the Dashboard page, click '[Download APIs and SDKs](https://developers.arcgis.com/downloads/data)' and go to the `Supplemental ArcGIS Runtime Data` tab.
+3. Click the download button next to `Projection Engine Data` to download projection engine data to your computer.
+4. Unzip the downloaded data on your computer.
+5. Create an `~/ArcGIS/Runtime/Data/PEDataRuntime` directory on your device and copy the files to this directory.
```

Comment:
it's possible we need to refine this further for Java: having a look at our `TransformationCatalog` class, there is guidance there for where the default location for such projection info is:

https://developers.arcgis.com/java/latest/api-reference/reference/com/esri/arcgisruntime/geometry/TransformationCatalog.html#getProjectionEngineDirectory()

Note that it is also possible to set the directory location:
https://developers.arcgis.com/java/latest/api-reference/reference/com/esri/arcgisruntime/geometry/TransformationCatalog.html#setProjectionEngineDirectory(java.lang.String)

in which case we'd need to say something like the following, but I'd like a second opinion from @colinanderson here first please to sanity check what I'm saying! 

```suggestion
5. Find the `<application directory>/resources/pedata` directory on your device and copy the files to this directory.
```